data.sql

```diff
@@ -1,5 +1,53 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO  animals (
+  name, 
+  date_of_birth, 
+  weight_kg
+)
+VALUES (
+    'Agumon',
+    '2020-02-03',
+    10.23
+);
+
+INSERT INTO  animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    weight_kg 
+)
+VALUES (
+    'Gabumon',
+    '2018-11-15',
+    2,
+    8
+);
+
+INSERT INTO  animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg 
+)
+VALUES (
+    'Pikachu',
+    '2021-01-07',
+    1,
+    false,
+    15.04
+);
+
+INSERT INTO  animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    weight_kg 
+)
+VALUES (
+    'Devimon',
+    '2017-05-12',
+    5,
+    11
+);
```

Comment:
- [ ] While inserting values the queries missing the fields(columns), Some of them missing the `neutered` and some `escape_attempts`. Please make sure that you insert the data into the table as requested in the project requirements. (If an animal never tried to escape, insert 0 for that animal in the `escape_atempts` column instead of the missing column).

![image](https://user-images.githubusercontent.com/83803414/183450150-a41088d2-5dd2-4f91-bd48-31da6e0c76bd.png)

Reply:
Thanks for your comments, added default values on the Create table statement

`CREATE TABLE animals (`
`   INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,`
`  name VARCHAR(100),`
`  date_of_birth DATE,`
`  escape_attempts INT DEFAULT 0,`
`  neutered BOOLEAN DEFAULT true,`
`  weight_kg decimal`
`);`

Reply:
@andgarzonmal, Good job. It will work as well. 👍